python/mp/effects/effect.py

```diff
@@ -33,11 +34,12 @@ def __init__(self, *args, **kwargs):
         self.color = kwargs.get('color')
         self.duration = kwargs.get('duration')
         self.delay = kwargs.get('delay', 0)
+        self.start_time = time.monotonic()
```

Comment:
this way every effect knows when it is instantiated. the idea is that we can use this for wall-clock timing, rather than counting frames. Maybe it should move into next() and be set the first time next() is called?